src/bun.js/bindings/bindings.cpp

```diff
@@ -4048,38 +4054,78 @@ class V8StackTraceIterator {
             return false;
         }
 
-        auto firstColon = line.find(':', openingParenthese + 1);
+        auto lineInner = StringView_slice(line, openingParenthese + 1, closingParenthese);
 
-        // if there is no colon, that means we only have a filename and no line number
+        {
+            auto marker1 = 0;
+            auto marker2 = lineInner.find(':', marker1);
 
-        //   at foo (native)
-        if (firstColon == WTF::notFound) {
```

Comment:
I find the comments that bread crumb where we are in the parsing of the stack trace helpful. The more the merrier. We should also add a note that if the stack trace format changes this is also needs updating. Or... can we comment referencing the V8 code we are mimicking.